### ISSUE TITLE
Packaging issues after [52cb294]

### ISSUE THREAD

**Issue by JustOff** [MoonchildProductions/UXP#1303]
After https://github.com/MoonchildProductions/UXP/commit/52cb294d2a5408d5eac6ac4ad5285165c6877932 I had the following issues when creating win32 package:

1) `mach package` results in an error:
```
 0:26.57 Error:
 0:26.57 Incorrect command line
 0:26.57 d:/UXP/toolkit/mozapps/installer/packager-uxp.mk:16: recipe for target 'make-archive' failed
 0:26.58 mozmake.EXE[1]: *** [make-archive] Error 7
 0:26.58 mozmake.EXE[1]: Leaving directory 'd:/UXP-build-palemoon/application/palemoon/installer'
 0:26.58 d:/UXP/application/palemoon/build.mk:9: recipe for target 'package' failed
 0:26.58 mozmake.EXE: *** [package] Error 2
 0:26.59 mozmake.EXE: Leaving directory 'd:/UXP-build-palemoon'
```

2) `mach mozpackage` does not produce a language pack

**Comment by mattatobin** [comment]
Would you mind giving me a verbose log? Also, what version of mozillabuild are you using?

**Comment by mattatobin** [comment]
What version of `Mozillabuild` are you using? Cause I do use a command line switch that was added in `7zip 15.05 beta` which is in `7zip 15.13` which ships with `Mozillabuild 2.2`.

`Mozillabuild 2.0` and `2.1` have `7zip 9.20`. So you will have to either manually upgrade 7zip or start using `Mozillabuild 2.2`.

As for the langpack issue I added a specific build target and mach command for it and reverted `mozpackage` to execute that target as it previously did.
